Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs

```diff
@@ -0,0 +1,140 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using HoloToolkit.Unity.InputModule;
+
+namespace HoloToolkit.Unity
+{
+#if UNITY_WSA && UNITY_2017_2_OR_NEWER
+    [RequireComponent(typeof(ControllerFinder))]
```

Comment:
Why do we require this component be attached to the GameObject if we're inheriting from that class?